safe-global/web-core
# fix: reduce threshold when removing owner

## What it solves

Resolves #1178

## How this PR fixes it

The `threshold` if the remaining number of owners is less than it.

## How to test it

Add two owners in the Safe creation flow and set the threshold to 2. Delete the second owner and observe the threshold change to 1.

## Screenshots

![threshold](https://user-images.githubusercontent.com/20442784/202249680-a393dd81-ebd8-4ecf-942f-e278567ecd86.gif)
